src/components/file-upload/file-upload.ts

```diff
@@ -174,11 +174,29 @@ export class MFileUpload extends ModulVue {
 
     private onOpen(): void {
         this.$emit('open');
+        // We need 2 nextTick to be able to have the wrap element in the DOM - MODUL-118
+        Vue.nextTick(() => {
+            Vue.nextTick(() => {
+                ['drag', 'dragstart', 'dragend', 'dragover', 'dragenter', 'dragleave', 'drop'].forEach(function(evt) {
+                    this.$refs.dialog.$refs.dialogWrap.addEventListener(evt, function(e) {
+                        e.preventDefault();
+                        e.stopPropagation();
+                    }.bind(this), false);
+                }.bind(this));
+            });
+        });
     }
 
     private onClose(): void {
         this.propOpen = false;
         this.$emit('close');
+        this.onCancelClick();
+        ['drag', 'dragstart', 'dragend', 'dragover', 'dragenter', 'dragleave', 'drop'].forEach(function(evt) {
+            this.$refs.dialog.$refs.dialogWrap.removeEventListener(evt, function(e) {
```

Comment:
es-tu certain que tes events sont bien débindés ici?  Pcq perso à moins d'utiliser une fonction de classe (removeEventListener('a', this.myMethod) ou une fonctione stockée dans une constante je me retrouve dans une situation d'event listener qui s'accumule sur mon élément...

après que le function règle ce problème? (pcq j'utilise normalement des arrow functions)

Reply:
Experts du téléverseur et drag n drop, je vous laisse approuver ou non cette modif.

Reply:
ouais, le removeEventListener est pas ok. Regarde dans la directive file drop

Reply:
C'est fait!